_posts/2020-06-07-Human-Curation-and-Digital-Datasets.md

```diff
@@ -0,0 +1,213 @@
+(DCC) of the [University of Edinburgh](https://www.ed.ac.uk/) is a
+department which is entirely devoted to the curation and preservation
+of digital artefacts.  Additionally, the [International Journal of
+Digital Curation](http://www.ijdc.net/) is one forum for discussion of
+these issues.  Other fora can be found on the DCC's [publication
+website](https://www.dcc.ac.uk/publications/ijdc).
+
+The second point is that corporate Data Governance is also a well
+known discipline which attempts to curate and standardise data within
+a corporate environment. The Data Steward ensures that the data can be
+both stored and handled securely in terms of law
+(e.g. [HIPAA](https://www.hhs.gov/hipaa/index.html) in The United
+States and [GDPR](http://data.europa.eu/eli/reg/2016/679/oj) in the
+European Union) and used profitably by the corporation.
+
+Both of these disciplines have much in common and often overlap in
+terms and structure.  In terms of IrishGen, Digital Curation is
+probably more important but both disciplines emphasise having data
+policies which guide the collection and organisation of data
+artefacts.  As stated by Eystein Thanisch, the primary policy of
+IrishGen is to take the original data seriously, even if that means,
+when working with digital editions like those on
+[CELT](https://celt.ucc.ie/), going back to the original manuscript
+source to check on possible transcription errors originating from the
+translation of the information from manuscript to printed source to
+digital source.
+
+Unfortunately, due to lack of resources and lack of time on the
+current curator's part, the insights from these two disciplines have
+not been explored in detail.  When creating databases like IrishGen,
+having a curator or a willing volunteer from one of these disciplines
+can be a boon to disrupting patterns of error before they reach a
+point at which an end user, such as a scholar or other interested party,
+encounters the data.
+
+## Automation
+
+Unlike the scribes and redactors of early medieval Ireland, who mainly
+relied on repetition and structure to reduce errors, IrishGen relies
+on those to create automated tools to translate digital versions of
+the genealogies, most prominently the CELT digital transcriptions,
+into [RDF](http://www.w3.org/TR/rdf11-primer/).
+
+However, automation is no panacea as even slight variations source
+material can cause the automation to introduce its own errors.  In
+IrishGen, a balance is struck between full automation and full manual
+with special case utilities that take some of the manual work out but
+leaving the integration to the curator.  In this case, there are two
+cases which are particularly easy for a computer to handle.  First, is
+the "String Pedigrees" ([Matthew
+Holmberg](http://isni.org/isni/0000000419902696), '[Towards a Relative
+Chronology of The Milesian Genealogical
+Scheme](https://dash.harvard.edu/handle/1/37945002)', pg. 18, for
+instance from LL's [Genelach Dal Copri
```

Comment:
Either say what LL is or hyperlink it. I didn't use that abbreviation in my post.

Reply:
Ah, right, sorry. I will sort it.